### PR TITLE
Components documentation versioning.

### PR DESCRIPTION
**Pull request by ghost**
Thank you for your contribution to MagicScript Project.
Before submitting this PR, please make sure:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [x] You have added unit tests
- [x] You have updated documentation

Description:

Docs versions page updated.
Docs for 2.0.2 generate based on: 2.0.2 tag (707aaa2).
Added docs-tag alias for docs tagging to `package.json`.

Screens:

### CODE REVIEW

File: website/versions.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[
+  "master",
```
Comment (kpiascik) [inline website/versions.json]: does this need to be master?  To me this implies nightly which we don't do.  Instead can we call it `latest` or something similar?  I'm open to other names.

**Review by kpiascik — APPROVED** [review]
LGTM, we can fix the updating of all of our example code separately.  Master is ok for now, but I'd like something different there as well.

File: website/versioned_docs/version-2.0.2/components/Audio.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+>
+
+```javascript
+import React from "react";
+import { View, Audio } from "magic-script-components";
+
+export default class MyApp extends React.Component {
+  render() {
+    return (
+      <View name="main-view">
```
[the remark this answers is not in the document]
Comment (SPaluchiewiczSofomo) [reply website/versioned_docs/version-2.0.2/components/Audio.md]: Yeah, I didn't include this yet into this PR as we didn't decide what's the better name for `Scene` - after this introducing new structure into the examples should be straight forward.

**Comment by leoz** [comment]
LGTM

**Review by leoz — APPROVED** [review]

**Pull request merged**